web/src/compose_state.ts

```diff
@@ -194,6 +194,13 @@ export function has_message_content(): boolean {
     return message_content() !== "";
 }
 
+export const MINIMUM_MESSAGE_LENGTH_TO_SAVE_DRAFT = 2;
+export function has_savable_message_content(): boolean {
+    return (
+        message_content() !== "" && message_content().length > MINIMUM_MESSAGE_LENGTH_TO_SAVE_DRAFT
```

Comment:
Also open to a shorter name for it to avoid this wrapping

Reply:
I think the `""` has a short length, so this doesn't need to wrap if you just delete that first redundant check.

Reply:
Oh of course 😅